README.md

```diff
@@ -301,7 +300,7 @@ In addition, each event contains the following headers:
 - `event_type` - the type of the event (see above)
 - `service` - the service that created the given playbook run
 - `status` - current of the playbook run
-- `account` - a legacy identifier for the account (tenant) the given playbook run belong to (replaced by `org_id`)
+- `account`* - a legacy identifier for the account (tenant) the given playbook run belongs to **(replaced by `org_id`, so this field may be null or absent)**
```

Comment:
I think we actually removed the field from the event interface altogether i.e. it will be always absent i.e. we can remove it from this doc